src/main/scala/maramafication/MaramaficationLoader.scala

```diff
@@ -0,0 +1,108 @@
+package maramafication
+
+import jawn.ast.{JArray, JValue}
+import maramafication.joint.{JointLocation, JointShape, Orientation}
+import util.Vector3
+
+import scala.io.Source
+
+case class MaramaficationLoader(_fileName: String) {
```

Comment:
A class docstring would be nice, especially to talk about how _fileName is handled.